dex-it/src/test/scala/com/wavesplatform/it/api/MatcherState.scala

```diff
@@ -5,13 +5,13 @@ import com.wavesplatform.dex.domain.account.KeyPair
 import com.wavesplatform.dex.domain.asset.AssetPair
 
 case class MatcherState(
-  offset: HttpOffset,
-  snapshots: HttpSnapshotOffsets,
-  orderBooks: Map[AssetPair, (HttpV0OrderBook, HttpMarketStatus)],
-  orderStatuses: Map[String, HttpOrderStatus],
-  orderTransactionIds: Map[String, Set[String]],
-  reservedBalances: Map[KeyPair, HttpBalance],
-  orderHistory: Map[KeyPair, Map[AssetPair, Seq[HttpOrderBookHistoryItem]]]
+                         offset: HttpOffset,
+                         snapshots: HttpSnapshotOffsets,
+                         orderBooks: Map[AssetPair, (HttpV0OrderBook, HttpOrderBookStatus)],
+                         orderStatuses: Map[String, HttpOrderStatus],
+                         orderTransactionIds: Map[String, Set[String]],
+                         reservedBalances: Map[KeyPair, HttpBalance],
+                         orderHistory: Map[KeyPair, Map[AssetPair, Seq[HttpOrderBookHistoryItem]]]
 ) {
```

Comment:
Probably a formatting is broken. Try to reformat the code

Reply:
fixed